app/assets/stylesheets/govuk-component/_search.scss

```diff
@@ -0,0 +1,184 @@
+    .js-enabled & {
+      .search-label {
+        position: absolute;
+        left: 15px;
+        top: 1px;
+        z-index: 1;
+        @include core-19($line-height: $input-size, $line-height-640: $input-size);
+        color: $secondary-text-colour;
+      }
+    }
+
+    .search-wrapper {
+      display: table;
+      width: 100%;
+      background: $white;
+    }
+
+    //general class applied to search input and button wrapper
+    .search-element {
+      position: relative;
+      display: table-cell;
+      vertical-align: top;
+    }
+
+    .search-input {
+      margin: 0;
+
+      // the .focus class is added by JS and ensures that the input remains above the label once clicked/filled in
+      &:focus,
+      &.focus {
+        z-index: 2;
+      }
+    }
+
+    .search-submit-wrapper {
+      width: 1%;
+    }
+
+    .search-submit {
+      position: relative;
+      padding: 0;
+      width: $input-size;
+      height: $input-size;
+      background-image: image-url("search-button.png");
+      background-repeat: no-repeat;
+      background-position: 2px 50%;
+      text-indent: -5000px;
+      overflow: hidden;
+
+      &:focus {
+        z-index: 2;
+      }
+
+      &::-moz-focus-inner {
+        border: 0;
+      }
+
+      @include device-pixel-ratio() {
+        background-size: 52.5px auto;
+        background-position: 115% 50%;
+      }
+    }
+  }
+
+  &.on-dark-background {
+    .search-label {
+      color: $white;
+    }
+
+    .search-submit {
+      background-color: $black;
+      color: $white;
+
+      &:hover {
+        background-color: lighten($black, 5%);
+      }
+    }
+
+    @if ($is-ie == false) or ($ie-version >= 8) {
+      .js-enabled & {
+        .search-label {
+          color: $secondary-text-colour;
+        }
+      }
+    }
+  }
+
+
+  &.on-light-background {
+    .search-label {
+      color: $black;
+    }
+
+    .search-input {
+      border: solid 1px $grey-2;
+    }
+
+    .search-submit {
+      background-color: $light-blue;
+      color: $white;
+
+      &:hover {
+        background-color: lighten($light-blue, 5%);
+      }
+    }
+
+    @if ($is-ie == false) or ($ie-version >= 8) {
+      .search-input {
+        border-right: 0;
+      }
+
+      .js-enabled & {
+        .search-label {
+          color: $secondary-text-colour;
+        }
+      }
+    }
+  }
+
+  &.search-large {
+    $input-size: 50px;
+
+    .search-label {
+      @include core-19($line-height: $input-size, $line-height-640: $input-size);
+    }
+
+    .search-input {
+      height: $input-size;
+    }
+
+    .search-submit {
+      width: $input-size;
+      height: $input-size;
+      background-position: 8px 50%;
+
+      @include device-pixel-ratio() {
+        background-size: 60px auto;
+        background-position: 160% 50%;
+      }
+    }
+  }
+
+  &.search-separate-label {
+    .search-label {
+      position: relative;
+      left: auto;
+
+      h1 {
```

Comment:
There was previously (see discussion of H1 above) but now there isn't, good spot, removed.